### PR TITLE
[ci] fix rllib release tests

### PR DESCRIPTION
**Pull request by can-anyscale**
We are now installing gynassium in ray-ml docker image and does not need to re-install it during release tests. Actually, re-installing them cause the test to fail.

Test:
- CI
- Release tests

### CODE REVIEW

**Review by ArturNiederfahrenhorst — APPROVED** [review]
@can-anyscale Thanks so much for digging into this! These dependencies are nasty!

**Comment by can-anyscale** [comment]
w00h00

**Comment by can-anyscale** [comment]
Oh I still need an approval from @aslonnie as code owners

**Review by aslonnie — APPROVED** [review]

**Pull request merged**
